generator/generator.go

```diff
@@ -146,17 +148,19 @@ func writeIfNotEmpty(b io.StringWriter, s string) {
 type codeOut struct {
 	// contents is the code that is contained in the output file.
 	contents string
+	// writeEvenIfEmpty forces the file to be written even with empty contents.
+	writeEvenIfEmpty bool
```

Comment:
Consider calling this `writeIfEmpty` to be a bit more pithy?

Reply:
Removed this struct.